surveys/templates/partials/sidebar.html

```diff
@@ -26,13 +26,13 @@ <h3><b>Just Spaces</b></h3>
                 </ul>
             </li>
             <li>
-                <a href="{% url 'surveys-list' %}">
+                <a href="{% url 'surveys-list' %}?published=f">
                     <i class="fas fa-pencil-alt icon edit"></i>
                     Edit survey
                 </a>
             </li>
             <li>
-                <a href="{% url 'surveys-list' %}">
+                <a href="{% url 'surveys-list' %}?published=t">
```

Comment:
What should get returned if the user visits `/surveys/` with no `published` param? I typically think of query parameters as offering optional filtering on top of a base view, but in this case it seems like the parameter is required in order to determine whether the view is for running or editing surveys.

If it makes sense to have a `/surveys/` endpoint that shows both published and unpublished surveys (something like `All Surveys`), then I think we should implement that endpoint as part of this PR. If that doesn't make sense, then I think we should reconsider the `published` param and instead refactor the URL hierarchy, e.g.:

- `/surveys/?published=t` -> `/surveys/run/`
- `/surveys/?published=f` -> `/surveys/edit/`

What do you think?